File: palidrome/palidrome.js
function palidrome(x){
    if(x<0){
        return false
    }
    let div = 1
    while(x == div*10){
        div*=10
    }
    while(x){
        let right = div%10
        let left = x / div
        if(left != right){
            return false
        }
        x = (div % div) / 10
        div = div / 100
    }
    return true
}

File: palidrome/palidrome.py
def palidrome(num):
    if num < 0 : return False
    div = 1
    while num >= 10 * div:
        div *=10
    while num:
        right = num % 10
        left = num // div
        if left != right: return False
        num = (num % div) //10
        div = div / 100
    return True
